roiextractors/segmentationextractor.py

```diff
@@ -238,7 +264,35 @@ def get_images(self):
                 -<image_name2>: <image_data2>
                 -<image_name3>: <image_data3>
         """
-        raise NotImplementedError
+        pass
+
+    def get_channel_names(self):
+        """
+        Names of channels in the pipeline
+        Returns
+        -------
+        _channel_names: list
+            names of channels (str)
+        """
+        return self._channel_names
+
+    def get_num_channels(self):
+        """
+        Number of channels in the pipeline
+        Returns
+        -------
+        no_of_channels: int
+        """
+        return len(self._channel_names)
+
+    def get_movie_location(self):
+        """
```

Comment:
Why does `SegmentationExtractor` need a path to the raw movie file location? That should be handled by `ImagingExtractor`.